src/routes/strategies/[strategy]/[status=positionStatus]-positions/[position]/+page.svelte

```diff
@@ -70,6 +70,16 @@
 			{/if}
 
 			<DataBox label="Highest value" value={formatDollar(getValueAtPeak(positionStats))} />
+
+			<DataBox label="Buy price">
```

Comment:
@AlexTheLion123 for "Buy" vs "Sell" labels, I would prefer if we add some logic so this doesn't break once we introduce short positions.

Elsewhere, we make this determination based on the trade's `planned_quantity`:

https://github.com/tradingstrategy-ai/frontend/blob/1ee41604c050884df93e6eadd56bed1a2220bf71/src/routes/strategies/%5Bstrategy%5D/%5Bstatus%3DpositionStatus%5D-positions/%5Bposition%5D/trade-%5Btrade%5D/%2Bpage.svelte#L16

This could be extracted to a `tradeType` helper that takes a `trade` and returns `Buy` or `Sell`.